src/api/WorkspaceConfig.ts

```diff
@@ -1,20 +1,23 @@
 import { ServiceDefinition } from '@scalecube/scalecube-microservice/lib/api';
 
-interface WorkspaceConfig {
-  name: string;
-  services: ServiceConfig[];
-  components: ComponentConfig[];
-}
-
-interface ServiceConfig {
+interface Service {
   serviceName: string;
   path: string;
   definition: ServiceDefinition;
   config: { [key: string]: any };
 }
 
-interface ComponentConfig {
+interface Component {
   componentName: string;
   path: string;
-  nodeSelector: string;
+  config: any;
```

Comment:
what should be included in this inner config? Should we put **nodeSelector** there?

Reply:
We don't need `nodeSelector` anymore as we will use the `id` of each item to render it.
This config can contain anything that the component will need. For example for Layout component it will include the config of nodes/elements/tabs that we we discussing there https://github.com/capsulajs/capsulahub-core/issues/4